### PR TITLE
minor improvements to cluster run process, download disch in m^3 s^-1 in config_to_data

### PR DESCRIPTION
**Pull request by aappling-usgs**

### CODE REVIEW

**Comment by aappling-usgs** [comment]
this test line fails locally (test-combine_ts.R#4):
```r
xy <- download_ts(c('suntime_calcLon', 'doobs_nwis', 'wtr_nwis', 'baro_nldas'), 'nwis_01467087', version='rds', on_local_exists="replace")
```
but that's just because Jordan is updating the ts files and so some of them are in an in-between state. e.g.:
```r
> sbtools::item_list_files(locate_ts('doobs_nwis', 'nwis_01467087'))$fname
[1] "ARCHIVE_20160716150432-nwis_01467087-ts_doobs_nwis.rds"   
[2] "ARCHIVE_20160421213413-nwis_01467087-ts_doobs_nwis.rds"   
[3] "ARCHIVE_20150617114401-nwis_01467087-ts_doobs_nwis.tsv.gz"
```

same with this one in test-get_ts$6:
```r
> doobs <- suppressWarnings(get_ts('doobs_nwis', 'nwis_06601200')) # verify_ts for doobs failed on test for units
Error in FUN(X[[i]], ...) : 
  found item but not file[s] in item nwis_06601200-ts_doobs_nwis (5581eca5e4b023124e8f3d5d): nwis_06601200-ts_doobs_nwis.rds
> sbtools::item_list_files(locate_ts('doobs_nwis', 'nwis_06601200'))$fname
[1] "ARCHIVE_20160716175001-nwis_06601200-ts_doobs_nwis.rds"   
[2] "ARCHIVE_20160419221324-nwis_06601200-ts_doobs_nwis.rds"   
[3] "ARCHIVE_20150617215446-nwis_06601200-ts_doobs_nwis.tsv.gz"
```

so those are fine.

**Comment by aappling-usgs** [comment]
appveyor:
```
* checking whether package 'mda.streams' can be installed ...Warning: running command '"c:/R/bin/i386/Rcmd.exe" INSTALL -l "C:/projects/mda-streams/mda.streams.Rcheck" --build --no-html --no-multiarch "C:\projects\MDA-ST~1\MDASTR~1.RCH\00_PKG~1\MDA~1.STR"' had status 1
 ERROR
Installation failed.
See 'C:/projects/mda-streams/mda.streams.Rcheck/00install.out' for details.
* DONE
```
and 00install.out has:
```
* installing *source* package 'mda.streams' ...
** R
** inst
** preparing package for lazy loading
Error : class "unitted_data.frame" is not exported by 'namespace:unitted'
ERROR: lazy loading failed for package 'mda.streams'
* removing 'C:/projects/mda-streams/mda.streams.Rcheck/mda.streams'
```
so yeah, do need unitted 0.2.7 after all

**Comment by aappling-usgs** [comment]
good! tests are finally failing on the right thing (get_ts and download_ts for doobs files being replaced)
```
  testthat results ================================================================
  OK: 93 SKIPPED: 0 FAILED: 2
  1. Error: combine_ts works (@test-combine_ts.R#4) 
  2. Error: get_ts works (@test-get_ts.R#6) 
  
  Error: testthat unit tests failed
  Execution halted
* checking PDF version of manual ... OK
* DONE
Status: 1 ERROR, 1 NOTE
```

File: DESCRIPTION [hunk 1]
```diff
@@ -20,7 +20,7 @@ Imports:
     geoknife (>= 1.2.0),
     sbtools (>= 1.0.2),
     streamMetabolizer (>= 0.9.7),
-    unitted (>= 0.2.6),
```
Comment (jread-usgs) [inline DESCRIPTION]: I think we also now have newer min versions of dataRet and geoknife (the versions after their https switchovers)

Comment (aappling-usgs) [inline DESCRIPTION]: oh, thanks. will add to next PR

**Pull request merged**